nowy.js

```diff
@@ -0,0 +1,198 @@
+let milliseconds = 0;
+let seconds = 0;
+let minutes = 0;
+let hours = 0;
+
+let leadingSeconds = 0;
+let leadingMinutes = 0;
+let leadingHours = 0;
+
+let timerInterval = null;
+let timerStatus = "stopped";
+
+let displayTimer = null;
+
+
+function stopWatch() {
+    milliseconds += 10
+    if(milliseconds/1000 === 1){
+    seconds++
+    milliseconds = 0;
+
+        if(seconds/60 === 1){
+        minutes ++;
+        seconds = 0;
+
+            if(minutes/60 === 1){
+            minutes = 0;
+            hours ++;
+            }
+        }
+    }
+    if(seconds < 10){
+        leadingSeconds = "0" + seconds.toString();
+      } else {
+        leadingSeconds = seconds;
+      }
+      if(minutes < 10){
+        leadingMinutes = "0" + minutes.toString();
+      } else {
+        leadingMinutes = minutes;
+      }
+      if(hours < 10){
+        leadingHours = "0" + hours.toString();
+      } else {
+        leadingHours = hours;
+      } 
+ displayTimer = document.querySelector('.timer').innerText = leadingHours + ':' + leadingMinutes + ':' + leadingSeconds + ":" + milliseconds;
+}
+
+const resetBtn = document.getElementById('reset-button');
+const splitBtn = document.getElementById('split-button');
+
+
+const startActionBtn = document.getElementById('start-button');
+
+
+
+const startBtn = document.querySelector('.startOrPause');
+const pauseBtn = document.getElementById('pause-button'); 
+startBtn.addEventListener('click', startTimer);
+pauseBtn.addEventListener('click', startTimer);
+
+function startTimer () {
+    if (timerStatus === "stopped") {
+    timerInterval = window.setInterval(stopWatch, 10);
+    document.getElementById('start-button').style.display = 'none';
+    document.getElementById('pause-button').style.display = 'block';
+    document.getElementById('reset-button').style.display = 'none';
+    document.getElementById('split-button').style.display = 'block';
+    timerStatus = "started"
+
+} else {
+
+    window.clearInterval(timerInterval);
+    document.getElementById('start-button').style.display = 'block';
+    document.getElementById('pause-button').style.display = 'none';
+    document.getElementById('reset-button').style.display = 'block';
+    document.getElementById('split-button').style.display = 'none';
+    timerStatus = "stopped";
+}};
+
+
+resetBtn.addEventListener('click', timeAndDataReset);
+
+function timeAndDataReset() {
+    milliseconds = 0;
+    seconds = 0;
+    hours = 0;
+    minutes = 0;
+    displayTimer = document.querySelector('.timer').innerText = '00:00:00:00';
```

Comment:
assign this value to a constant and define it on top on the file so it can be easily configured.
Use uppercase for config variables
```
const INIT_TIMER_VALUE = '00:00:00:00'
```